tests/apiv2/import_data.py

```diff
@@ -0,0 +1,2513 @@
+                "template_uuid": "688c46fb-5edb-40a3-8273-1af7923e2215",
+                "template_version": "15",
+                "timestamp": "1690190991",
+                "uuid": "f11692e1-d32d-4b3f-8cd1-2b99c04b72f6"
+            }
+        ],
+        "Org": {
+            "id": "1",
+            "name": "CERT-FR",
+            "uuid": "56bdf779-46f8-4353-bdf9-2bb95bce2212"
+        },
+        "Orgc": {
+            "id": "1",
+            "name": "CERT-FR",
+            "uuid": "56bdf779-46f8-4353-bdf9-2bb95bce2212"
+        },
+        "RelatedEvent": [],
+        "ShadowAttribute": [],
+        "Tag": [
+            {
+                "colour": "#f89595",
+                "exportable": True,
+                "hide_tag": False,
+                "id": "40",
+                "name": "fr-classif:non-classifiees=\"NON-CLASSIFIEES\"",
+                "numerical_value": None,
+                "user_id": "0"
+            },
+            {
+                "colour": "#ffffff",
+                "exportable": True,
+                "hide_tag": False,
+                "id": "9254",
+                "name": "cert-fr:TLP=\"clear\"",
+                "numerical_value": None,
+                "user_id": "0"
+            },
+            {
+                "colour": "#ffffff",
+                "exportable": True,
+                "hide_tag": False,
+                "id": "9253",
+                "name": "cert-fr:PAP=\"clear\"",
+                "numerical_value": None,
+                "user_id": "0"
+            },
+            {
+                "colour": "#007350",
+                "exportable": True,
+                "hide_tag": False,
+                "id": "473",
+                "name": "cert-fr:fiabilite=\"Bonne\"",
+                "numerical_value": None,
+                "user_id": "0"
+            }
+        ],
+        "analysis": "0",
+        "date": "2023-07-24",
+        "disable_correlation": False,
+        "distribution": "0",
+        "extends_uuid": "",
+        "id": "17909",
+        "info": "[CERT-FR] [ANSSI] FIN21 : un groupe cybercriminel aux multiples rançongiciels",
+        "locked": False,
+        "org_id": "1",
+        "orgc_id": "1",
+        "proposal_email_lock": False,
+        "publish_timestamp": "0",
+        "published": False,
+        "sharing_group_id": "0",
+        "threat_level_id": "4",
+        "timestamp": "1690272052",
+        "uuid": "64be448c-53f0-4e74-a53d-4c3d0abe1822"
+    }
+}   
+        cls.path_json = os.path.join(os.path.dirname(__file__),'misp_event.json')
+        with open(cls.path_json,'w') as misp_event:
+            json.dump(cls.data,misp_event)
+
+    def test_import_misp(self):
+        logging.info('Test import misp')
+        with open(self.path_json, 'rb') as fichier:
+            files = {'misp_file_json': (self.path_json, fichier)}
+
+            r=client.post('/api/v2/import_data/import_misp_json',files=files)
+            self.assertEqual(r.status_code,200)
+            
```

Comment:
It woudl also be good to assert that the observables that are created are properly formed (right values, right neighbors, etc.)